e2e-playwright/smoke-tests-suite/panels.spec.ts

```diff
@@ -29,22 +29,32 @@ test.describe(
         return win.grafanaBootData?.settings?.panels ?? {};
       });
 
+      const vizPicker = dashboardPage.getByGrafanaSelector(selectors.components.PanelEditor.toggleVizPicker);
+
       // Loop through every panel type and ensure no crash
       for (const [_, panel] of Object.entries(panelTypes)) {
         if (panel.hideFromList || panel.state === 'deprecated') {
           continue; // Skip hidden and deprecated panels
         }
 
-        // Select the panel type in the viz picker
-        const vizPicker = dashboardPage.getByGrafanaSelector(selectors.components.PanelEditor.toggleVizPicker);
-        await vizPicker.click();
-        await dashboardPage.getByGrafanaSelector(selectors.components.PluginVisualization.item(panel.name)).click();
+        try {
+          // Select the panel type in the viz picker
+          await page.waitForTimeout(500);
```

Comment:
in general, adding a timeout like this is just doomed to become flaky again - we'd rather make Playwright wait for an element on the screen or something if we can. Do we have any idea what the flaky failures were here? If it was that the vizPicker wasn't clickable, we could do this, which makes Playwright wait for the element to be in the DOM before proceeding

```suggestion
          await expect(vizPicker).toBeVisible();
```

Reply:
I can update to that. Locally is was that the ds modal was not closing in time, 60000ms exceeded, race conditions - viz picker existed but not ready to be clicked, moving too fast between panels and click was failing.